crm/server_settings.py

```diff
@@ -2,7 +2,7 @@
 import sentry_sdk
 from sentry_sdk.integrations.django import DjangoIntegration
 
-DEBUG = False
```

Comment:
DEBUG = False is the dajngo default [read more](https://django.doctor/advice/C4005)